.github/workflows/helm-release.yaml

```diff
@@ -0,0 +1,20 @@
+name: helm-release
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+
+jobs:
+  call-update-helm-repo:
+    uses: grafana/helm-charts/.github/workflows/update-helm-repo.yaml@main
+    with:
+      charts_dir: production/helm
```

Comment:
I wonder whether this will also release `promtail` and `fluent-bit`.

Reply:
yes, my guess is it will. I think we could either a) remove everything but the README from those charts, and include the SHA they were deleted at (in a separate PR that we merge before this one), or b) can we sepecify a `charts_dir` of `production/helm/loki`?

I actually prefer option `a` as I think it cleans up the code base a bit.

Reply:
https://github.com/grafana/loki/pull/6978 <- PR removing the deprecated helm charts